java/lite/pom.xml

```diff
@@ -232,6 +232,7 @@
                     <exclude>TypeRegistryTest.java</exclude>
                     <exclude>UnknownEnumValueTest.java</exclude>
                     <exclude>UnknownFieldSetLiteTest.java</exclude>
```

Comment:
not new, but why are we excluding the lite test here? Worth a look.

Reply:
Hmm, good question. I'm not sure why.